module-apps/application-phonebook/widgets/InputLinesWithLabelWidget.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include "application-phonebook/data/PhonebookInternals.hpp"
+#include "application-phonebook/data/PhonebookItemData.hpp"
+#include "application-phonebook/widgets/ContactListItem.hpp"
+
+#include <ListItem.hpp>
+#include <Text.hpp>
+#include <TextFixedSize.hpp>
+
+namespace gui
+{
+    class InputLinesWithLabelWidget : public ContactListItem
```

Comment:
This is not new file -> GH is stupid i just made small changes here (with name fix...).